pkg/server/runtime/runtime.go

```diff
@@ -247,7 +256,13 @@ func (s *SingularityRuntime) ListContainerStats(ctx context.Context, req *k8s.Li
 
 // UpdateRuntimeConfig updates the runtime configuration based on the given request.
 func (s *SingularityRuntime) UpdateRuntimeConfig(ctx context.Context, req *k8s.UpdateRuntimeConfigRequest) (*k8s.UpdateRuntimeConfigResponse, error) {
-	glog.Warningf("Ignoring runtime config update %v", req)
+	config := req.GetRuntimeConfig()
+	if config == nil {
+		return &k8s.UpdateRuntimeConfigResponse{}, nil
+	}
+	if s.networkManager != nil && config.NetworkConfig.PodCidr != "" {
```

Comment:
`networkManager` is never nil, unless you don't call `NewSingularityRuntime` which is nonsense. 
also just in case `config.GetNetworkConfig().PodCidr`

Reply:
Right !